cirq/iqm/iqm_device.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IQM devices https://iqm.fi/devices"""  # TODO: PQC-5
+
+from typing import cast
+import cirq
+from cirq import devices, ops
+
+
+class Adonis(devices.Device):
+    """IQM's five-qubit superconducting device with pairwise connectivity.
+    Details: https://iqm.fi/devices
```

Comment:
Add the details in this docstring. The link should just be a reference.

Reply:
I tried to write an ASCII art style diagram on the qubit connectivity but couldn't get it to render sensibly in typical docstring viewing situations, hence the link. OTOH there is now the class constant `QUBIT_DIAGRAM` which may server the purpose.

@vtomole What sort of details would you prefer to have here? There are many many things we can disclose about our unit but most of them are probably of no interest to Cirq users...

Reply:
The idea is that cirq users will be using cirq (perhaps through an IDE) or viewing from readthedocs (https://cirq.readthedocs.io/en/latest/).  They may not follow a link.  These users will need basic information about the device and whether it is what they are looking to use.  For detailed information, they can follow up in the link, but they will need basic information on what this class is.

Reply:
Added documentation here: https://github.com/quantumlib/Cirq/pull/2701/commits/c98e112b6ae252d6f167706231e6c37d41dedb2b